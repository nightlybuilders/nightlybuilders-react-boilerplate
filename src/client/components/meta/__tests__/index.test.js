import React from 'react'
import { shallow } from 'enzyme'

import { Meta } from '../'

const ChildMeta = () => <meta charSet="utf-8" />

describe('Component/Meta', () => {
  const props = {
    description: 'Test Description',
    title: 'Test Title',
  }

  test('should render without throwing an error', () => {
    const wrapper = shallow(<Meta {...props} />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should render children', () => {
    const wrapper = shallow(
      <Meta {...props}>
        <ChildMeta />
      </Meta>,
    )
    expect(wrapper.find(ChildMeta)).toHaveLength(1)
  })
})
