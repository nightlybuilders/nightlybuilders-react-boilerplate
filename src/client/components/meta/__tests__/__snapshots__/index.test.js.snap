// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Meta should render without throwing an error 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <title>
    Test Title
  </title>
  <meta
    content="Test Description"
    name="description"
  />
</HelmetWrapper>
`;
